### ISSUE TITLE
convert the analogue meter to temperature meter

### ISSUE THREAD

**Issue by lrtal** [G6EJD/ESP8266-Analogue-Meter#1]
hello, is it possible to convert the analogue meter to  temperature meter (analogue)?

**Comment by G6EJD** [comment]
Yes, but it depends on the sensor you intend to use, if analogue and measuring a DC voltage as a function of temperature then the line 

109   input_value = input_value / 100; 

would simply need to be adjusted to display temperature. So the ADC input range is 0-1023
perhaps:
109   input_value = input_value / 100 * 0.95;

If using a digital temp sensor then the result would most likely be in degrees and
108   temp = readsensor();
109   input_value = temp / 5; // the / 5 bit needs to be fine tuned.

**Comment by lrtal** [comment]
nice, is see. think i will try it out with digital temp sensor
